msteams-platform/qsg-bot-configuration-experience.yml

```diff
@@ -57,12 +57,17 @@ items:
 
      1. Select **Clone**.
 
+     > [!div class="nextstepaction"]
+     > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Set+up+local+environment&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fqsg-bot-configuration-experience%3Ftutorial-step%3D2&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fqsg-bot-configuration-experience.yml&documentVersionIndependentId=c2ff51a0-e012-ef6c-9eb2-5e74e8ec7a4a&author=surbhigupta&metadata=*+ID%253A+e473e1f3-69f5-bcfa-bcab-54b098b59c80+%250A*+Service%253A+%2A%2Amsteams%2A%2A)
```

Comment:
Update contentURL like this:
https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/qsg-bot-configuration-experience.yml#L47

Update for all others as well.